tools/postinstall

```diff
@@ -7,6 +7,13 @@ patch_package() {
     node_modules/.bin/patch-package
 }
 
+# Since we take react-native from Git, some packages that ship with it
+# and are used in the build need their build scripts run.
+build_rn_packages() {
+    ( cd node_modules/react-native/packages/react-native-codegen/ && \
+        yarn clean && node scripts/build.js )
```

Comment:
Given the `set -e`, this doesn't need `&&`:
```suggestion
    (
        cd node_modules/react-native/packages/react-native-codegen/
        yarn clean
        node scripts/build.js
    )
```

(Or maybe the latter two commands can stay in `&&` form to stick closer to the `yarn build` script.)

Then a comment mentioning that this is based on the package's `yarn build` would be helpful. Otherwise it's left a bit mysterious why we do these particular steps.